templates/delivery/form.html.twig

```diff
@@ -116,11 +116,18 @@
         </div>
       {% endif %}
     </div>
-    <div class="col-md-5">
-      <button id="delivery-submit" type="submit" class="btn btn-block btn-lg btn-primary h-100">
+    <div class="col-md-5 delivery__form__buttons">
+      <button id="delivery-submit" type="submit" class="btn btn-block btn-lg btn-primary">
         <i class="fa fa-spinner fa-spin hidden" id="loader"></i>
         {% trans %}basics.save{% endtrans %}
       </button>
+      {% if is_granted('ROLE_ADMIN') and not is_new and delivery.order is not empty %}
+        <a href="{{ path('admin_store_delivery_new',
+          { id: delivery.store.id, frmrdr: delivery.order|hashid(16) }
```

Comment:
`delivery.store` is only set for "lastmile" deliveries. This fields is empty for foodtech

Reply:
Which lead to this error for foodtech deliveries
![image](https://github.com/coopcycle/coopcycle-web/assets/5613584/e7e0cfe2-225c-43c5-99f7-a8688bd78bb2)

Reply:
Fixed in 5ef7b02f81b01894d4031360b8d65a0561da98d7